NetCore/LoadResizeSave.cs

```diff
@@ -1,55 +1,52 @@
-﻿using BenchmarkDotNet.Attributes;
 using System;
 using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
 using System.IO;
 using System.Linq;
-
-using ImageMagick;
+using BenchmarkDotNet.Attributes;
 using FreeImageAPI;
+using ImageMagick;
 using PhotoSauce.MagicScaler;
-using SkiaSharp;
-
-using System.Drawing;
-using System.Drawing.Drawing2D;
-using System.Drawing.Imaging;
-using SystemDrawingImage = System.Drawing.Image;
-
-using SixLabors.ImageSharp.Processing;
 using SixLabors.ImageSharp.Formats.Jpeg;
+using SixLabors.ImageSharp.Processing;
+using SkiaSharp;
 using ImageSharpImage = SixLabors.ImageSharp.Image;
 using ImageSharpSize = SixLabors.Primitives.Size;
+using SystemDrawingImage = System.Drawing.Image;
 
 namespace ImageProcessing
 {
-    [MemoryDiagnoser]
     public class LoadResizeSave
     {
-        const int ThumbnailSize = 150;
-        const int Quality = 75;
-        const string ImageSharp = nameof(ImageSharp);
-        const string SystemDrawing = nameof(SystemDrawing);
-        const string MagickNET = nameof(MagickNET);
-        const string FreeImage = nameof(FreeImage);
-        const string MagicScaler = nameof(MagicScaler);
-        const string SkiaSharpCanvas = nameof(SkiaSharpCanvas);
-        const string SkiaSharpBitmap = nameof(SkiaSharpBitmap);
+        private const int ThumbnailSize = 150;
```

Comment:
I hit ctrl K + E at some point and didn't notice until much later.

Reply:
That's fine. I don't mind it at all :D